HEC.FDA.ViewModel/AggregatedStageDamage/CalculatedStageDamageVM.cs

```diff
@@ -753,7 +754,17 @@ private void WriteDetailsCsvFile(ScenarioStageDamage scenarioStageDamage, List<U
         {
             try
             {
-                List<string> stageDamageDetails = scenarioStageDamage.ProduceStructureDetails();
+                //Its necessary for the details to be written with names for the impact areas, not FIDs. It helps users troubleshoot. I don't like this, but that's why the dictionary is passed in. to translate at the
+                //lowest level before it's written
+                Dictionary<int, string> iaNames = [];
+                ImpactAreaElement iaEle = StudyCache.GetChildElementsOfType<ImpactAreaElement>()[0];
+                List<ImpactAreaRowItem> iaRows = iaEle.ImpactAreaRows;
+                for(int i = 0; i < iaRows.Count; i++)
+                {
+                    iaNames[i] = iaRows[i].Name;
+                }
+                
+                List<string> stageDamageDetails = scenarioStageDamage.ProduceStructureDetails(iaNames);
                 List<string> damagedElementCounts = UncertainPairedData.ConvertDamagedElementCountToText(quantityDamagedElementsUPD);
                 List<string> stageDamageText = UncertainPairedData.ConvertFunctionsToText(stageDamageFunctions);
```

Comment:
I think we can remove this from our reporting, it was intended to support our understanding of uncertainty in damage, I think it will cause confusion without a lot of support.

Reply:
done